Sources/MarkupRenderer.swift

```diff
@@ -19,10 +19,21 @@ public final class MarkupRenderer {
 		self.baseFont = baseFont
 	}
 
-	public func render(text: String) -> NSAttributedString {
-		let elements = MarkupParser.parse(text: text)
-		let attributes = [NSAttributedString.Key.font: baseFont]
+	public func render(text: String, foregroundColor: UIColor? = nil, backgroundColor: UIColor? = nil) -> NSAttributedString {
+		var elements = MarkupParser.parse(text: text)
+		elements = elements.map {
+			switch $0 {
+			case .strong(let children): if children.count == 0 { return MarkupNode.text("**") }
+			case .emphasis(let children): if children.count == 0 { return MarkupNode.text("__") }
+			case .delete(let children): if children.count == 0 { return MarkupNode.text("~~") }
```

Comment:
Can you replace `children.count` with `children.isEmpty`?

Reply:
done